core/templates/dev/head/css/oppia.css

```diff
@@ -4981,3 +4981,19 @@ md-card.preview-conversation-skin-supplemental-card {
   padding: 0;
   width: 1px;
 }
+
+.modal-dialog md-checkbox .md-icon {
```

Comment:
Remember to localize this one too! (You will probably need to add extra classes to the html attribute)

Reply:
Done !

Reply:
Done!